### PR TITLE
Added tooltip constant file to store text / tsx content for tooltip across the app.

### PR DESCRIPTION
**Pull request by onehanddev**
- Implement Scorecard Tab Content with Dynamic Descriptions
- Improves code maintainability by centralizing descriptions in the tooltip.tsx file.

### CODE REVIEW

File: client/src/components/ui/info-button.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import * as React from "react";
 
 import { InfoIcon } from "lucide-react";
 
+import { cn } from "@/lib/utils";
```
Comment (andresgnlez) [inline client/src/components/ui/info-button.tsx:6]: No need of importing `cn` if we are not mixing classes

File: client/src/components/ui/info-button.tsx [hunk 2]
```diff
@@ -16,8 +18,10 @@ import {
 export default function InfoButton({
   title,
   children,
+  classNames,
 }: PropsWithChildren<{
   title?: string;
+  classNames?: string;
```
Comment (andresgnlez) [inline client/src/components/ui/info-button.tsx:24]: Better `className` so we are consistent with the rest of the codebase.

**Comment by andresgnlez** [comment]
@onehanddev 
image

 There's a double scrollbar in the dialog of the overview table.

**Comment by onehanddev** [comment]
> @onehanddev image
> 
> There's a double scrollbar in the dialog of the overview table.

Ahh, this seems to be one of those screen issues, Can you please share the resolution of your screen, as I cant reproduce it in my screen size with zoomed in/out enabled.

**Comment by onehanddev** [comment]
> > @onehanddev image
> > There's a double scrollbar in the dialog of the overview table.
> 
> Ahh, this seems to be one of those screen issues, Can you please share the resolution of your screen, as I cant reproduce it in my screen size with zoomed in/out enabled.

The latest commit should fix the issue. Can you please check again @andresgnlez

**Pull request merged**